.github/workflows/run_doublet-detection.yml

```diff
@@ -24,7 +24,9 @@ on:
       - main
     paths:
       - analyses/doublet-detection/**
-      - .github/workflows/run_doublet-detection.yml
+      - "!analyses/doublet-detection/Dockerfile"
+      - "!analyses/doublet-detection/.dockerignore"
+      # - .github/workflows/run_doublet-detection.yml
```

Comment:
I had actually added this line in at one point, see here for some context: https://github.com/AlexsLemonade/OpenScPCA-analysis/issues/676

If you want to keep this line deleted for now and come back to this later, I'm fine with that!

Reply:
Yes, I know why it is there, but it would make this PR never pass, as it would keep trying to use the current docker image, not the one defined by the changes to the Dockerfile.

Reply:
Right, totally understand that, sorry if I wasn't clear!  - I'm _only_ saying if you want to wholesale delete, rather than comment out, that would be ok too.